src/Modules/SimplCommerce.Module.Catalog/Areas/Catalog/Controllers/BrandApiController.cs

```diff
@@ -29,7 +29,7 @@ public async Task<IActionResult> Get()
         {
             var brandList = await _brandRepository.Query().Where(x => !x.IsDeleted).ToListAsync();
 
-            return Json(brandList);
+            return new JsonResult(brandList);
```

Comment:
I think it should be.

  ```cs     
        [HttpGet]
        public async Task<ActionResult<IEnumerable<Brand>>> Get()
        {
            var brandList = await _brandRepository.Query().Where(x => !x.IsDeleted).ToListAsync();
            return brandList;
        }
```

And in the case of return an anonymous class. We have to create a view model for it.

Reply:
Great idea @thiennn !

Reply:
@thiennn It is strange that Api Controllers cannot be decorated with [ApiController]! Have you any idea about it? I have already tested it but it fails.